nilearn/tests/test_cache_mixin.py

```diff
@@ -17,6 +15,15 @@
 from nilearn._utils.testing import assert_raises_regex
 
 
+def _get_subdirs(top_dir):
+    top_dir = pathlib.Path(top_dir)
+    children = list(top_dir.glob("*"))
+    return [child for child in children if child.is_dir()]
+
+
+def _get_n_subdirs(top_dir):
```

Comment:
I think it's better to use `len(_get_subdirs(top_dir))` instead of another function.